runatlantis.io/docs/server-side-repo-config.md

```diff
@@ -522,6 +522,7 @@ If you set a workflow with the key `default`, it will override this.
 | Key         | Type              | Default | Required   | Description                                             |
 |-------------|-------------------|---------|------------|---------------------------------------------------------|
 | users       | []string          | none    | yes        | list of github users that can approve failing policies  |
```

Comment:
It seems that users is not required? This is not this PR changes, so please ignore if you have not enough time.

Reply:
Agreed, fixed.